src/classes/HMSSDK.tsx

```diff
@@ -1288,6 +1290,47 @@ export class HMSSDK {
     return await HMSManager.getSessionMetaData({ id: this.id });
   };
 
+  getVideoTrackLayerDefinition = async (remoteVideoTrack: HMSRemoteVideoTrack) => {
+    logger?.verbose('#Function getVideoTrackLayerDefinition', {
+      id: this.id,
+      remoteVideoTrack
+    });
+
+    if(Platform.OS === 'android') {
+      const layerDefinition = await HMSManager.getVideoTrackLayerDefinition({
+        id: this.id,
+        trackId: remoteVideoTrack.trackId
+      });
+
+      console.log(layerDefinition);
+
+      return HMSHelper.getSimulcastLayersWithCurrentActive(layerDefinition);
+    } else {
+      console.log('API currently not available for iOS');
+      return 'API currently not available for iOS';
+    }
+  }
+
+  setVideoTrackLayer = async (remoteVideoTrack: HMSRemoteVideoTrack, layer: HMSLayer) => {
```

Comment:
These functions are to be added in `HMSRemoteVideoTrack` class not on the `HMSSDK` instance

```kotlin
fun setLayer(HMSLayer: HMSLayer)
        
fun getLayerDefinition() : List<HMSSimulcastLayerDefinition>

fun getLayer() : HMSLayer
```